cmd/prober/prober.go

```diff
@@ -258,6 +285,18 @@ func observeRequest(host string, r ReadProberCheck) error {
 	return nil
 }
 
+func observeGrcpGetTrustBundleRequest(ctx context.Context, fulcioGrpcClient fulciopb.CAClient) error {
+	s := time.Now()
+
+	_, err := fulcioGrpcClient.GetTrustBundle(ctx, &fulciopb.GetTrustBundleRequest{})
+	latency := time.Since(s).Milliseconds()
+	if err != nil {
+		return err
+	}
+	exportGrpcDataToPrometheus(0, "grpc://"+fulcioGrpcURL, "GetTrustBundle", "GET", latency)
```

Comment:
gRPC is different than HTTP in that response status codes are a part of `error` when it returns a 4xx/5xx error. Can we log the code using something like https://github.com/sigstore/fulcio/blob/main/pkg/server/grpc_server_test.go#L126-L128 instead of `0`?

You'll need to also not return early when there is an error.